instaclone/src/App.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import dummyData from './dummy-data';
+import PostContainer from './components/PostContainer/PostContainer';
+import './App.css';
+import SearchBar from './components/SearchBar/SearchBar';
+
+class App extends React.Component {
+  state={
+    data: dummyData
+  }
+
```

Comment:
Nicely done... I like that you keep your data in App level. This will help you later when you need to filter post based on user input.